indico/web/forms/fields.py

```diff
@@ -716,3 +719,20 @@ def _value(self):
 class IndicoEnumRadioField(IndicoEnumSelectField):
     widget = JinjaWidget('forms/radio_buttons_widget.html', orientation='horizontal', single_kwargs=True)
     option_widget = RadioInput()
+
+
+class IndicoProtectionField(IndicoEnumRadioField):
+    widget = JinjaWidget('forms/protection_message_widget.html', single_kwargs=True)
+    radio_widget = JinjaWidget('forms/radio_buttons_widget.html', orientation='horizontal', single_kwargs=True)
+
+    def __init__(self, *args, **kwargs):
+        kwargs['enum'] = ProtectionMode
+        super(IndicoProtectionField, self).__init__(*args, **kwargs)
+
+    def render_protection_message(self):
```

Comment:
Not an issue, I just think this makes the code a bit prettier. Hope I don't piss you off!
```python
    def render_protection_message(self):
        protected_object = self.get_form().protected_object
        parent_type = _('Event') if isinstance(protected_object.protection_parent, Event) else _('Session')
        rv = render_template('_protection_info.html', field=self, protected_object=protected_object,
                             parent_type=parent_type)
        return Markup(rv)
```